### ISSUE TITLE
Korean Support

### ISSUE THREAD

**Issue by sleeping-player** [rayshift/translatefgo#21]
First, Thank you for making this.

Maybe I'm going to make several separate issues here, all to ask if it's possible to implement Korean translation stories in other server clients. This is because Korean FGO publishers recently caused a very unpleasant incident that did not keep their promises, and users want to move to JP and NA server.

There are already a lot of scripts that users have translated by themselves based on Japanese servers, and if possible, it seems that the script will be ported by users through the github project commit method.

There will be many problems, but I will first ask if "this app" can implement a Korean script.

**Comment by neobenedict** [comment]
It is possible, but DeepL has no KR support. I'm a little bit against putting the official scripts in because of copyright reasons. The NA scripts are already pushing the boundaries there, and adding KR scripts would give a definite reason for people to move causing lost revenue to KR and legal risk for myself. If there is significant interest I would consider it.

The app itself is only a tool to download the replacement scripts, it doesn't contain any code related to building scripts.

**Comment by sleeping-player** [comment]
https://gall.dcinside.com/typemoon/1557175
OK. Like the link above, community translations are accumulating. These were translated shortly after the original story of JP server was released, and cannot be the official scripts of KR server (and NA), which runs two years after the Japanese version.

If so, is there a way to create a replacement script?

**Comment by neobenedict** [comment]
It's done in google spreadsheets since you need to match the text up 1:1 and include some things like facial expressions via tags, for example https://i.imgur.com/ZM29ZQr.png

The other problem is the in-game font for JP does not support korean so you would need a modded APK. I am developing one for RU at the moment.

**Comment by sleeping-player** [comment]
Is the Modded apk's name betterfgo or is it another mod?

**Comment by neobenedict** [comment]
> Is the modded apk's name betterfgo or is it another mod?

It isn't released yet, betterfgo was a test for the underlying framework (il2cpp hooking library).

**Comment by sleeping-player** [comment]
This would be a very complicated task. I think I saw that the schedule is for February, that's why it takes time..

**Comment by sleeping-player** [comment]
I paid for Patreon to see some translations in other languages ​​that were recently added.
I heard that Russian language support will be added, has this solved the font problem?
And I think I need some guidance on the pre-release key, can this be contacted by Discord?

**Comment by neobenedict** [comment]
The new languages haven't been added yet, they should be available soon.

**Comment by jesee030** [comment]
> First, Thank you for making this.
> 
> Maybe I'm going to make several separate issues here, all to ask if it's possible to implement Korean translation stories in other server clients. This is because Korean FGO publishers recently caused a very unpleasant incident that did not keep their promises, and users want to move to JP and NA server.
> 
> There are already a lot of scripts that users have translated by themselves based on Japanese servers, and if possible, it seems that the script will be ported by users through the github project commit method.
> 
> There will be many problems, but I will first ask if "this app" can implement a Korean script.

we need chinese too,the Chinese publisers did some sick things too Orz

**Comment by neobenedict** [comment]
I don't think this will be possible as the korean/simplified chinese font isn't supported and there is no way to add new fonts without a modified game.

**Comment by NonameverX** [comment]
DEEPL now supports Korean! Can you add Korean to the rayshift translator?

**Comment by NonameverX** [comment]
DEEPL now supports Korean! Can you add Korean to the rayshift translator?

**Comment by biue1230** [comment]
> I don't think this will be possible as the korean/simplified chinese font isn't supported and there is no way to add new fonts without a modified game.

The deepl now supports Korean

I want to use the Korean translation function in .depl even if I don't bring the Korean server's unique translation. Other games also have Korean servers, and even if I use the Korean translation program, I have never raised a problem until now. Examples.. Girls' Front 2, Umamusme PrettyDerby, etc..

I'm a Korean user, but I do a Japanese server, especially since it's not translated to understanding game mechanics, I hope the program is supported.

**Comment by neobenedict** [comment]
Reevaluating this, seems the font works better now.

Still need a small team of people to import the translations though, and permissions from the authors of the OC4/etc translations.

![Image](https://github.com/user-attachments/assets/d7342114-27a5-4b29-b4cc-32e730cdc891)
